yarn.lock

```diff
@@ -4,14 +4,14 @@
 
 "@ampproject/remapping@^2.1.0":
   version "2.1.2"
-  resolved "https://registry.yarnpkg.com/@ampproject/remapping/-/remapping-2.1.2.tgz#4edca94973ded9630d20101cd8559cedb8d8bd34"
+  resolved "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.1.2.tgz"
```

Comment:
Why this change is nessesary? I'd prefer to keep `registry.yarnpkg.com` with package hashes.